apps/storybook/src/Alert.stories.tsx

```diff
@@ -165,15 +165,34 @@ export const Sticky: Story<AlertProps> = (args) => {
 
 Sticky.args = {
   children: 'This is sticky!',
-  clickableText: 'More Info.',
   type: 'informational',
   isSticky: true,
 };
 
 export const Empty: Story<AlertProps> = (args) => {
   return (
     <Alert type='informational' {...args}>
-      {args.children}
+      <Alert.Icon />
+      <Alert.Message>This is an empty info message.</Alert.Message>
+    </Alert>
+  );
+};
+
+Empty.args = {
```

Comment:
Two `Empty.args` declarations in this file

Reply:
This has been changed to `CustomIcon.args` and the `children` prop has changed